docs/usage_examples.md

```diff
@@ -0,0 +1,88 @@
+### Usage Examples
+
+Foyer supports atomtyping of both all-atom and coarse-grained molecular systems, and
+also allows for separate force fields to be used to atom-type separate portions of a
+molecular system.
+
+#### Creating a box of ethane
+Here we use mBuild to construct a box filled with ethane molecules and use Foyer to
+atom-type the system, applying the OPLS force field, and save to run-able GROMACS 
+files.
+```python
+import mbuild as mb
+from mbuild.examples.ethane.ethane import Ethane
+
+ethane_box = mb.fill_box(Ethane(), 100, box=[2, 2, 2])
+ethane_box.save('ethane-box.gro')
+ethane_box.save('ethane-box.top', forcefield_name='oplsaa')
+```
+----
+#### Creating a box of coarse-grained ethane
+Again we will use mBuild to construct a box filled with ethane molecules.  However,
+now we will model ethane using a united-atom description and apply the TraPPE force
+field during atom-typing.  Note how particle names are prefaced with an underscore so
+that Foyer knows these particles are non-atomistic.
+```python
+import mbuild as mb
+
+class CH3_UA(mb.Compound):
+    def __init__(self):
+        super(CH3_UA, self).__init__()
+        self.add(mb.Particle(name='_CH3'))
+        self.add(mb.Port(anchor=self[0], separation=0.075), 'up')
+
+ethane_UA = mb.Compound()
+ethane_UA.add(CH3_UA(), 'ch3_1')
+ethane_UA.add(CH3_UA(), 'ch3_2')
+mb.force_overlap(ethane_UA['ch3_1'], ethane_UA['ch3_1']['up'],
+                 ethane_UA['ch3_2']['up'])
```

Comment:
Might be worth adding a CG example to mbuild so we don't need to spend time here explaining how to build this since it's not really pertinent to foyer itself. Or maybe the class could be simplified to something like:

```python
ethane_UA = mb.Compound()
ch3_1 = mb.Particle(name='_CH3', pos=[0, 0, 0])
ch3_2 = mb.Particle(name='_CH3', pos=[0.2, 0, 0])
ethane_UA.add([ch3_1, ch3_2])
ethane_UA.add_bond((ch3_1, ch3_2))
```